frontend/src/scenes/insights/Insight.scss

```diff
@@ -51,6 +51,11 @@
             }
         }
 
+        .overflow-hidden {
```

Comment:
We probably shouldn't override a class like this. Is it necessary? I'm pretty sure it has no effect because the wrapping class (`insights-graph-container`) has been removed

Reply:
Scratch that. This does have a purpose but we still shouldn't do it. Looking into an alternative